src/components/Sidebar.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { slide as Menu } from "react-burger-menu";
+
+
+class Sidebar extends React.Component {
+  constructor(props)  {
+      super(props);
+      this.state = {
+          isOpen: false
+      }
+      this.isMenuOpen = this.isMenuOpen.bind(this);
+     }; 
+
+     isMenuOpen () {
```

Comment:
This function name makes me think "Is my Menu open ?", not "I want to toggle my menu"
You should rename this "toggleMenu", it'd be clearer